boxes/static/javascripts/box_list.js

```diff
@@ -2,24 +2,27 @@ $(document).ready(function() {
   /*
     On click copy the box link to the clipboard
   */
+  $(".box-options").click(function(){
+    $(this).submit();
+  });
   $(".copy-to-clipboard-js").on("click", function(){
     $this = $(this);
-    var previousText = $this.text();
+    var previousText = $this.html();
```

Comment:
On this line and the following lines, why do you need `html()` instead of `text()`?